interfaces/interfaces.go

```diff
@@ -2,7 +2,43 @@ package interfaces
 
 type IWritter interface {
 	WriteToDisk() error
+	CheckIfExists() error
+	DeletePath() error
+	GetPath() string
 }
 
 // TODO:
 // add objects that satisfy the interface
+
+func Write(paths []IWritter) error {
```

Comment:
please leave in this file only the interface definition   
you can make of course an object or just have a helpers file

Reply:
addressed in fa38b125828f208631c482c74923ffb3fbe4f1da

or 

https://github.com/TudorHulban/GoLayouter/blob/fa38b125828f208631c482c74923ffb3fbe4f1da/domain/interfaces/interfaces.go#L3